e2e-tests/cypress/support/page-objects/Discover.ts

```diff
@@ -22,7 +22,7 @@ export class Discover {
     cy.get('[data-test-subj=discoverSaveButton]').click();
     cy.get('[data-test-subj=savedObjectTitle]').type(reportName);
     cy.get('[data-test-subj=confirmSaveSavedObjectButton]').click({ force: true });
-    cy.contains('was saved', { timeout: 10000 }).should('exist');
+    cy.contains('was saved', { timeout: 60000 }).should('exist');
```

Comment:
60 seconds sounds big here, it means that when for some reason the test fails, the test will wait an extra 50 seconds to fail because Cypress will try to retry the check. If there is a non-determinism here, maybe we need to intercept specific API response and then, check if this button exist?